src/components/DarkModeToggle.tsx

```diff
@@ -20,7 +22,13 @@ export const DarkModeToggle = (props: {
 
   return (
     <label
-      className={`ToolIcon ToolIcon_type_floating ToolIcon_size_M`}
+      className={clsx(
+        "ToolIcon ToolIcon_type_floating ToolIcon_size_M",
+        "toggle-appearance",
```

Comment:
lets not render the button when the prop is false instead of controlling via css. Since host will either control it completely or not control it.

Reply:
sounds good